main/group_vars/all

```diff
@@ -9,4 +9,4 @@ sync_node: ""
 sync_node_comm_user: "root"
 slack_hook: ""
 trust_rpc1: "rpc-mainnet.commercio.network:80"
-trust_rpc2: "rpc2-mainnet.commercio.network:80"
\ No newline at end of file
+trust_rpc2: "rpc2-mainnet.commercio.network:80"
```

Comment:
The rpcs data should be customizable selecting the chain. It should be explain in the README or better download from chains repository. You can add some variable if you want.